pkg/ebpf/tracer_test.go

```diff
@@ -1547,7 +1547,6 @@ const (
 func testDNSStats(t *testing.T, domain string, success int, failure int, timeout int, serverIP string) {
 	config := NewDefaultConfig()
 	config.CollectDNSStats = true
-	config.CollectLocalDNS = true
```

Comment:
Why was this removed?

Reply:
This was added before unnecessarily since none of the tracked connections are local.